src/commands/openCodeUsagesView/registerOpenUsagesViewCommand.ts

```diff
@@ -0,0 +1,32 @@
+import * as vscode from 'vscode'
+import { OPEN_USAGES_VIEW_PREFIX } from './constants'
+import { CodeUsageNode } from '../../views/usages/UsagesTree/CodeUsageNode'
+import { UsagesTreeProvider } from '../../views/usages'
+
+export async function registerOpenUsagesViewCommand(
+    context: vscode.ExtensionContext,
+    variableKey: string,
+    codeUsagesView: vscode.TreeView<CodeUsageNode>,
+    usagesDataProvider: UsagesTreeProvider
+) {
+    const commandId = OPEN_USAGES_VIEW_PREFIX + variableKey
```

Comment:
could we not just pass the variable key as an argument to the command, rather than registering a new command for each variable?